### DISCUSSION TITLE
where is the server side map/pin data stored?

### DISCUSSION THREAD

**Discussion by Nikolay1243** [Mydayyy/Valheim-ServerSideMap#31] [Q&A] [answered]
I need to back it up

**Comment by Mydayyy** [comment, marked as answer]
Greetings,

the file is stored in the same location as the world.

![image](https://user-images.githubusercontent.com/8339771/112159713-bd082300-8be9-11eb-8adb-32d2f4986c2d.png)

Nameing is as follows: WORLDNAME.mod.serversidemap.explored

Both map and pin data are in that file.

Cheers
Mydayyy

**Reply by Nikolay1243** [reply]
ah thanks :)
